app/services/watsonSpeech.js

```diff
@@ -0,0 +1,39 @@
+const TextToSpeechV1 = require("ibm-watson/text-to-speech/v1");
+const {IamAuthenticator} = require("ibm-watson/auth");
```

Comment:
`{ IamAuthenticator }` would be better.

Reply:
source?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import

Reply:
`import {myExport} from '/modules/my-module.js';`
Source: your link